### PR TITLE
1.2 feature/admin project details

### PR DESCRIPTION
**Pull request by danbergelt**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by BDesselle — DISMISSED** [review]

**Review by agordon1123 — DISMISSED** [review]
👍

**Review by armandoroman1016 — DISMISSED** [review]

File: mission-control-client/src/components/dashboard/admin-dashboard/DashboardMetrics.js [hunk 1]
```diff
@@ -28,4 +30,7 @@ const mapStateToProps = state => {
   };
 };
 
-export default connect(mapStateToProps, null)(DashboardMetrics);
+export default connect(
+  mapStateToProps,
+  null
```
Comment (dionys93) [inline mission-control-client/src/components/dashboard/admin-dashboard/DashboardMetrics.js:35]: `null` is basically an empty object, but I wonder which is considered the convention.

File: mission-control-client/src/components/dashboard/admin-dashboard/DashboardProject.js [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
\ No newline at end of file
+const mapStateToProps = state => {
+  return {
+    activeProductStore: state.activeProductStore
+  };
+};
+
+export default connect(
+  mapStateToProps,
+  {}
```
Comment (dionys93) [inline mission-control-client/src/components/dashboard/admin-dashboard/DashboardProject.js]: `{}` or `null`?

**Review by dionys93 — CHANGES_REQUESTED** [review]
Consistency across react-redux `connect`. It looks like `null` is the convention:
https://react-redux.js.org/api/connect

**Review by dionys93 — APPROVED** [review]

**Review by agordon1123 — APPROVED** [review]

**Review by BDesselle — APPROVED** [review]

**Pull request merged**